packages/web/src/features/clubDetails/services/getMyClub.ts

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import apiReg006 from "@sparcs-clubs/interface/api/registration/endpoint/apiReg006";
+
+import { useQuery } from "@tanstack/react-query";
+
+import { z } from "zod";
+
+import {
+  axiosClientWithAuth,
+  defineAxiosMock,
+  UnexpectedAPIResponseError,
+} from "@sparcs-clubs/web/lib/axios";
+
+import mockupRegistraion from "./_mock/mockupRegistraion";
+
+import type { ApiReg006ResponseOk } from "@sparcs-clubs/interface/api/registration/endpoint/apiReg006";
+
+export const useGetMyClub = () =>
+  useQuery<ApiReg006ResponseOk, Error>({
+    queryKey: [apiReg006.url()],
+
+    queryFn: async (): Promise<ApiReg006ResponseOk> => {
+      const { data, status } = await axiosClientWithAuth.get(
+        apiReg006.url(),
+        {},
+      );
+      switch (status) {
+        case 200:
+          return apiReg006.responseBodyMap[200].parse(data);
+        default:
+          throw new UnexpectedAPIResponseError();
+      }
+    },
+  });
+
+defineAxiosMock(mock => {
+  mock.onGet(apiReg006.url()).reply(() => {
+    const dummy: z.infer<(typeof apiReg006.responseBodyMap)[200]> =
+      mockupRegistraion;
+    return [200, dummy];
```

Comment:
```suggestion
    return [200, mockupRegistraion];
```
이렇게 해도 될 것 같은데 확인해주세요!